packages/itwinui-css/backstop/tests/menu.html

```diff
@@ -617,72 +617,87 @@ <h3>Header</h3>
-                <div class="iui-description">0x0123456789</div>
-              </span>
-              <svg-caret-down-small
-                class="iui-button-icon"
-                aria-hidden="true"
-              ></svg-caret-down-small>
-            </button>
-
-            <svg-chevron-right
-              class="iui-chevron"
-              aria-hidden="true"
-            ></svg-chevron-right>
-
-            <!-- IModel Button -->
-            <button
-              class="iui-button iui-borderless iui-active iui-large iui-header-button iui-header-dropdown-button"
-              type="button"
-              aria-label="Model picker"
-              aria-expanded="true"
-              aria-haspopup="true"
-              aria-current="location"
-              id="demo-header-imodel-button"
-            >
-              <svg-model
-                class="iui-button-icon iui-header-button-icon"
-                aria-hidden="true"
-              ></svg-model>
-              <span class="iui-button-label">
-                <div>Model Beta</div>
-                <div class="iui-description">0x0123456789</div>
-              </span>
-              <svg-caret-down-small
-                class="iui-button-icon"
+          <nav
+            class="iui-header-breadcrumbs"
+            aria-label="Breadcrumbs"
+          >
+            <ol class="iui-header-breadcrumbs-list">
+              <li class="iui-breadcrumbs-item">
+                <span class="iui-header-breadcrumb-button-wrapper">
+                  <button
+                    class="iui-header-breadcrumb-button"
+                    type="button"
+                    aria-label="Project picker"
+                    aria-expanded="false"
+                    id="test-button-1"
+                  >
+                    <img
+                      class="iui-header-breadcrumb-button-icon"
+                      src="../assets/image-test.jpg"
+                      draggable="false"
+                    />
+                    <span class="iui-header-breadcrumb-button-text">
+                      <div class="iui-header-breadcrumb-button-text-label">Project Alpha</div>
+                      <div class="iui-header-breadcrumb-button-text-sublabel">0x0123456789</div>
+                    </span>
+                    <svg-caret-down-small
+                      class="iui-header-breadcrumb-button-dropdown-icon"
+                      aria-hidden="true"
+                    ></svg-caret-down-small>
+                  </button>
+                </span>
+              </li>
+              <li
+                class="iui-breadcrumbs-separator"
                 aria-hidden="true"
-              ></svg-caret-down-small>
-            </button>
+              >
+                <svg-chevron-right aria-hidden="true"></svg-chevron-right>
+              </li>
+              <li class="iui-breadcrumbs-item">
+                <span
+                  class="iui-header-breadcrumb-button-wrapper"
```

Comment:
I am seeing some inconsistency in the various html files. Can you verify and document the correct html structure?

Reply:
Good catch, I've updated `skip-to-content.html` & `menu.html` to match the latest HTML structure in `header.html`.